website/content/en/docs/config/ai/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: AI
+weight: 90
+---
+
+There are two kinds of scenario to use Lima with AI:
+
+- [AI in Lima](./ai-in-lima): running an AI agent inside a VM
+- [Lima in AI](./lima-in-ai): calling Lima's MCP tools from an AI agent running outside a VM
```

Comment:
I would call this "MCP tools in Lima".

Because "Lima in AI" means to me that the AI tool itself will run `limactl` commands on the host, and not via MCP.

Reply:
"Agent inside Lima" and "Agent outside Lima" might be more easily comparable?